aip/general/0180.md

```diff
@@ -139,6 +139,99 @@ code, as such changes will be seen as breaking by those users.
 whether a proposed change is likely to break users, and an expansive reading of
 this guidance could ostensibly prevent _any_ change (which is not the intent).
 
+#### Default values must not change
+
+Default values are the values set by servers for resources when they are not
+specified by the client. This section only applies to static default values within
+fields on resources and does not apply to dynamic defaults such as the default IP
+address of a resource.
+
+Changing the default value is considered breaking and **must not** be done. The
+default behavior for a resource is determined by its default values, and this
+**must not** change across minor versions.
+
+For example:
+
+```proto
+message Book {
+    string name = 1;
+    // The genre of the book
+    // If this is not set when the book is created, the field will be given a value of FICTION.
+    enum Genre {
+      UNSPECIFIED = 0;
+      FICTION = 1;
+      NONFICTION = 2;
+    }
+}
+```
+
+```proto
```

Comment:
This second Book message should be removed, right?

Reply:
Yep! Removing